tier1/{{cookiecutter.project_slug}}/.github/workflows/checks.yml

```diff
@@ -0,0 +1,69 @@
+name: "run-linting-checks"
+on:
+  pull_request:
+    branches: [main, dev]
+
+
+jobs:
+  resolve-repolinter-json:
+    uses: DSACMS/repo-scaffolder/.github/workflows/extendJSONFile.yml@add-repolinter-workflows
```

Comment:
Looks like `extendJSONFile.yml` is on dev and not main. For this PR, could you merge into dev? I'll do a merge to main after